domains/omkhalane.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Documentation website for is-a.dev",
+  "repo": "https://github.com/is-a-dev/docs",
+  "owner": {
+    "username": "omkhalane",
+    "email": "[om.khalane.dev@gmail.com]"
```

```suggestion
    "email": "om.khalane.dev@gmail.com"
```
Brackets are not a valid character for the email key.